clusters/app.ci/supplemental-ci-images/openvpn.yaml

```diff
@@ -0,0 +1,43 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    namespace: ci
+    name: openvpn
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: openvpn
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: openvpn:latest
+    source:
+      dockerfile: |
+        FROM registry.ci.openshift.org/openshift/centos:stream8
```

Comment:
nit: This won't cover any other architecture, unfortunately. :(

/lgtm
/approve